resources/create_archive.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Creates an archive suitable for distribution (standard layout for binaries,
+# libraries, etc.).
+
+set -e
+
+if [ -z ${PACKAGE_TITLE} -o -z ${VERSION} -o -z ${DESTDIR} ]; then
+    echo "PACKAGE_TITLE, VERSION, and DESTDIR environment variables must be set."
```

Comment:
can we we move everything to a common packaging branch?

Reply:
Not for this PR, but with my other work on the packaging makefiles I've identified a lot of duplication that we can get rid of.